package.json

```diff
@@ -4,8 +4,7 @@
   "description": "Exercises for JS1 Week 3",
   "license": "CC-BY-SA-4.0",
   "scripts": {
```

Comment:
Just checking if the way we would run jest tests locally would change after this? And if so, should the README.md be updated accordingly?